gnd/src/main/java/com/google/android/gnd/persistence/sync/LocalMutationSyncWorker.java

```diff
@@ -126,4 +136,12 @@ private Mutation incrementRetryCount(Mutation mutation, Throwable error) {
         .setLastError(error.toString())
         .build();
   }
+
+  private void sendNotification(UploadProgress uploadProgress) {
```

Comment:
Should we add an interface for this method--or make it static and have it live in the Notification manager (we'd just have to make it take the R.string as a second argument)? 

It's more or less the same in the other worker (tiles).

Reply:
8d75cd8